pallets/schemas/src/mock.rs

```diff
@@ -93,3 +144,14 @@ pub fn new_test_ext() -> sp_io::TestExternalities {
 	ext.execute_with(|| System::set_block_number(1));
 	ext
 }
+
+/// Create and return a simple test AccountId32 constructed with the desired integer.
+pub fn test_public(n: u8) -> AccountId32 {
```

Comment:
we really need this in some sort of overall test utils crate.  Not this PR though.

Reply:
I added it to this mock.rs which is okay, right?  You are just saying that eventually, we need to create a common crate for this, correct?